research/src/pages/select/select.proposal.mdx

```diff
@@ -46,7 +46,7 @@ then select the appropriate one for you.
 | [`type`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/type)                       | `string`                | `"select-one"`                        | Returns either `"select-multiple"` or `"select-one"`                                                                                                     |     |
 | `validationMessage`                                                                                     | `string`                | `""`                                  | Represents the message that describes the validation constraints that the control does not satisfy                                                       |     |
 | `validity`                                                                                              | `ValidityState`         |                                       | Represents the current validity state of the control                                                                                                     |     |
-| `value`                                                                                                 | `string`                | `""`                                  | Returns the `value` property of the first selected `<option>` if there is one, otherwise an empty string                                                 |     |
+| `value` <a id="value"></a>                                                                              | `string`                | `""`                                  | Returns the `value` property of the first selected `<option>` if there is one, otherwise an empty string                                                 |     |
```

Comment:
What's the purpose of this `<a>`?  I didn't see any visible change on the deploy preview.

Reply:
it's to provide an anchor for linking to, for example: `[selected](#selected) prop <br/> [value](#value) prop`

Reply:
Ah, I didn't see the referencing anchor, thanks.